Xiaomi BLE: Document behaviour for new entities (#24306)
home-assistant/home-assistant.github.io: source/_integrations/xiaomi_ble.markdown

- Smoke
- Moisture

> The entities for the sensor classes are added only after the values are actually received. This means entities for values that are broadcasted at an lower interval (e.g. battery) might show up later.

Reviewer: ```suggestion
The entities for the sensor classes are added after the values are first received. This means entities for values that are broadcasted at a lower interval (e.g., battery) might show up later.
```
